opentelemetry-sdk/CHANGELOG.md

```diff
@@ -1,7 +1,13 @@
 # Changelog
 
+## Deprecation Notice
+
+XrayIdGenerator has been migrated to the opentelemetry-aws crate starting from 0.10.0. [#33](https://github.com/open-telemetry/opentelemetry-rust-contrib/pull/33)
```

Comment:
the message is not very clear. Could you rephrase so that it is clear that we are referrung to 0.10.0 of the aws create.

Also, this can be under a "##Removed" section.

Reply:
I guess has been deprecated in this crate, not removed yet.

Reply:
It can however be moved inside of the version. I think this may have been inspired by my jaeger deprecation, but in this case we're only deprecating a component. Where as for jaeger we're deprecating the crate. 

Could be:

```
# Changelog

## vNext

### Deprecated
...

```

Reply:
how about this?
`
### Deprecated

- XrayIdGenerator in the opentelemetry-sdk has been deprecated and moved to version 0.10.0 of the opentelemetry-aws crate.
`